packages/calcite-components/src/components/tile/tile.scss

```diff
@@ -3,46 +3,52 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
+ * @prop --calcite-tile-accent-color-press: Specifies the color of the component's selection indicators (radio, checkbox, border).
```

Comment:
@eriklharper Have some doc recommendations for consistency across all of the properties, which look to extend beyond the PR:

> * @prop --calcite-tile-accent-color-press: When the parent `calcite-tile-group` has a `selectionMode` that is not `"none"`, specifies the color of the component's selection elements, such as the radio, checkbox, and border.
> * @prop --calcite-tile-background-color: Specifies the component's background color.
>  * @prop --calcite-tile-border-color: Specifies the component's border color.
>  * @prop --calcite-tile-corner-radius: Specifies the component's corner radius.
> * @prop --calcite-tile-heading-text-color: Specifies the component's `heading` text color.
>  * @prop --calcite-tile-shadow: Specifies the shadow around the component.
 > * @prop --calcite-tile-text-color: Specifies the component's `description` and `icon` text color, but not the `heading` text color.

Reply:
Updated! :)